docs/getting-started-checklist.md

```diff
@@ -0,0 +1,25 @@
+# Scarf - Getting Started Checklist

```

Comment:
```suggestion
# Getting Started Checklist

```

Reply:
These heading end up in the table of contents so we don't need to mention "Scarf" in all of them